spec/convection/model/template/validate_bytesize_spec.rb

```diff
@@ -21,13 +21,13 @@ class Convection::Model::Template
           description 'Validations Test Template - Excessive Bytesize'
 
           200.times do |count|
-            resource "EC2_INSTANCE_#{count}" do
-              type 'AWS::EC2::Instance'
-              property 'AvailabilityZone', 'us-east-1a'
+            ec2_instance "EC2_INSTANCE_#{count}" do
+              availability_zone 'us-east-1a'
               property 'ImageId', 'ami-76e27e1e'
               property 'KeyName', 'test'
```

Comment:
These can be replaced with the existing methods vs. re-defining the properties.

**side note**: Should we warn when re-defining properties as well?